### PR TITLE
Issue/123 gae auth

### PR DESCRIPTION
**Pull request by iperdomo**

### CODE REVIEW

**Comment by jonase** [comment]
@iperdomo The `akvo.flow-services.geoshape-export` namespace still uses user/pwd auth.

File: src/akvo/flow_services/stats.clj [hunk 1]
```diff
@@ -28,17 +28,19 @@
...
 
-(defn datastore-spec [server usr pwd]
-  {:server server
-   :email usr
-   :password pwd
-   :port 443})
+(defn datastore-spec [server]
+  (let [host (first (str/split server #"\."))
+        cfg (@config/configs host)]
+    {:hostname server
```
Comment (jonase) [inline src/akvo/flow_services/stats.clj]: should this be `host` instead of `server`?

Comment (iperdomo) [inline src/akvo/flow_services/stats.clj]: OK, thanks will change that...

**Comment by jonase** [comment]
:+1: Code reviewed.

**Pull request merged**